CMakeLists.txt

```diff
@@ -444,6 +444,16 @@ if(__BUILD_FOR_R)
   endif(MSVC)
 endif(__BUILD_FOR_R)
 
+# fmtlib/fmt
+#if (NOT BUILD_STATIC_LIB)
+#  OPTION(BUILD_SHARED_LIBS "Build shared libs" ON)  # fmt reads this flag.
+#endif(NOT BUILD_STATIC_LIB)
+#SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -funroll-loops")
```

Comment:
Though it is not commented out, I should say that I find the flag `-funroll-loops` suspect. It is rarely used, rarely useful. Though it has its uses, one should carefully document it.

Reply:
Removing it entirely would be great.

Reply:
(I had a typo which was "it is not commented"=>"it is now commented".)